### ISSUE TITLE
convert result mismatch between command-line  and python_api

### ISSUE THREAD

**Issue by ChiFang** [tensorflow/tensorflow#17501]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Linux Ubuntu 16.04
- **TensorFlow installed from (source or binary)**:
binary: by "pip3 install --upgrade tensorflow-gpu"
- **TensorFlow version (use command below)**:
tensorflow-gpu (1.6.0)
- **Python version**: 
Python 3.5.2
- **Bazel version (if compiling from source)**: 0.11.0
- **GCC/Compiler version (if compiling from source)**:
gcc (Ubuntu 5.4.0-6ubuntu1~16.04.9) 5.4.0 20160609
- **CUDA/cuDNN version**: CUDA 9.0 cuDNN 7
- **GPU model and memory**: Null
- **Exact command to reproduce**:

### Describe the problem
 I save my pre-trained model into .pb file according to [freeze_graph.py](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/tools/freeze_graph.py) and [optimize_for_inference.py](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/tools/optimize_for_inference.py)
Once I covert to .tflite by command-line with following cmd: 
> ~/tensorflow/bazel-bin/tensorflow/contrib/lite/toco/toco \
--input_file=RD_Net_TransConv_fuse.pb \
--output_file=./RD_Net_TransConv_fuse.tflite \
--input_format=TENSORFLOW_GRAPHDEF \
--output_format=TFLITE \
--inference_type=FLOAT \
--input_data_types =FLOAT \
--input_arrays=Placeholder \
--output_arrays=seq_4/Conv/BiasAdd \
--input_shapes=1,240,320,3 \
--logtostderr

the error show that "TransposeConv" is not supported : 
> 2018-03-07 15:07:49.317477: F tensorflow/contrib/lite/toco/tflite/export.cc:304] Some of the operators in the model are not supported by the standard TensorFlow Lite runtime. If you have a custom implementation for them you can disable this error with --allow_custom_ops. Here is a list of operators for which you will need custom implementations: TransposeConv.

If I use [python_api](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/toco/g3doc/python_api.md) to convert my tensor into .tflite, it work fine without any error msg.
But the .tflite can not be used in android....  it will crash when  I want to getOutputName or run

error msg in android:
> Can't get Output Names, model error:Invalid handle to Interpreter.

does tensorflow lite support "TransposeConv"? I don't see any information in [Compatibility Guide](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/g3doc/tf_ops_compatibility.md)

Any suggestion? Thanks~~~

**Comment by aselle** [comment]
1. If you don't see an operator in the README, it is likely not supported. TransposeConv is not yet supported. 

2. Could you please give the python call that you use to generate the model, so we can see if it is effectively the same. The interfaces are a little different (Python trying to be more intuitive), but its possible you are using them slightly differently.

**Comment by ChiFang** [comment]
Hi @aselle 
I write a sub function to convert as following:
```
def Save_lite(sess, input_tensors, net, pbName="TFLite_converted_model", clear_devices = True):
    print("----")
    print(net.name)
    print("----")
    
    output_node_names = net.name
    output_node_names = output_node_names.split(":")
    output_node_names = output_node_names[0]
    input_graph_def = net.graph.as_graph_def()
    
    # Remove all the explicit device specifications for this node. This helps to
    # make the graph more portable.
    if clear_devices:
        for node in input_graph_def.node:
            node.device = ""
    
    
    frozen_graphdef = graph_util.convert_variables_to_constants( 
        sess,
        input_graph_def,
        output_node_names.split(",")
    )

    output_graph_def = graph_util.remove_training_nodes(frozen_graphdef)
    out_tensors = [net]
    tflite_model = tf.contrib.lite.toco_convert(output_graph_def, [input_tensors], out_tensors)
    open(pbName+".tflite", "wb").write(tflite_model)
```

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aselle: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by ianholing** [comment]
Is there a timing for add "TransposeConv" to the TFLITE compatible operations? I have the same problem exporting my model to tflite.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @andrehentz: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @andrehentz: It has been 45 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @andrehentz: It has been 60 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by ianholing** [comment]
True! Thanks I will check that again as soon as I can.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @andrehentz: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @andrehentz: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
